specifyweb/frontend/js_src/lib/components/Notifications/hooks.tsx

```diff
@@ -5,10 +5,12 @@ import { ajax } from '../../utils/ajax';
 import { formatDateForBackEnd } from '../../utils/parser/dateFormat';
 import type { IR, RA } from '../../utils/types';
 import { sortFunction } from '../../utils/utils';
+import { MINUTE, SECOND } from '../Atoms/timeUnits';
 import { formatUrl } from '../Router/queryString';
 import type { GenericNotification } from './NotificationRenderers';
 
-const INITIAL_INTERVAL = 5000;
+const INITIAL_INTERVAL =
+  process.env.NODE_ENV === 'development' ? Number(MINUTE) : 5 * SECOND;
```

Comment:
MINUTE is already a number, so no need to convert it?

```suggestion
  process.env.NODE_ENV === 'development' ? MINUTE : 5 * SECOND;
```

Reply:
Sorry about late reply--ESLint/Prettier wanted to make that change, I had it w/o Number before 😅 we're all good to change it back